Bigtable/src/BigtableClient.php

```diff
@@ -0,0 +1,190 @@
+     *     @type array $keyFile The contents of the service account credentials
+     *           .json file retrieved from the Google Developer's Console.
+     *           Ex: `json_decode(file_get_contents($path), true)`.
+     *     @type string $keyFilePath The full path to your service account
+     *           credentials .json file retrieved from the Google Developers
+     *           Console.
+     *     @type float $requestTimeout Seconds to wait before timing out the
+     *           request. **Defaults to** `0` with REST and `60` with gRPC.
+     *     @type int $retries Number of retries for a failed request.
+     *           **Defaults to** `3`.
+     * }
+     * @throws GoogleException If the gRPC extension is not enabled.
+     */
+    public function __construct(array $config = [])
+    {
+        $this->requireGrpc();
+        $config += [
+            'scopes' => [
+                self::FULL_CONTROL_SCOPE,
+                self::ADMIN_SCOPE
+            ],
+            'projectIdRequired' => true
+        ];
+        $this->connection = new Grpc($this->configureAuthentication($config));
+        $this->setLroProperties(new LongRunningConnection($this->connection), [
+            [
+                'typeUrl' => 'type.googleapis.com/google.bigtable.admin.instance.v2.UpdateInstanceMetadata',
+                'callable' => function ($instance) {
+                    $name = InstanceAdminClient::parseName($instance['name'])['instance'];
+                    return $this->instance($name, $instance);
+                }
+            ], [
+                'typeUrl' => 'type.googleapis.com/google.bigtable.admin.instance.v2.CreateInstanceMetadata',
+                'callable' => function ($instance) {
+                    $name = InstanceAdminClient::parseName($instance['name'])['instance'];
+                    return $this->instance($name, $instance);
+                }
+            ]
+        ]);
+    }
+
+    /**
+     * Create a new instance.
+     *
+     * Example:
+     * ```
+     * $operation = $bigtable->createInstance('my-instance');
+     * ```
+     *
+     * @codingStandardsIgnoreStart
+     * @see https://cloud.google.com/bigtable/docs/reference/admin/rpc/google.bigtable.admin.v2#CreateInstanceRequest CreateInstanceRequest
+     *
+     * @param string $instanceId The instance ID.
+     * @param array $options [optional] {
+     *     Configuration options
+     *
+     *     @type string $displayName **Defaults to** the value of $instanceId.
+     *     @type int $instanceType  Possible values include `Instance_Type::PRODUCTION` and `Instance_Type::DEVELOPMENT`.
+     *           **Defaults to**`Instance_Type::TYPE_UNSPECIFIED`.
+     *     @type array $labels For more information, see
+     *           [Using labels to organize Google Cloud Platform resources](https://cloudplatform.googleblog.com/2015/10/using-labels-to-organize-Google-Cloud-Platform-resources.html).
+     *     @type array $clusters [] {
```

Comment:
What do you think about representing these as a separate object? `Google\Cloud\Bigtable\Cluster`. I see there are RPCs which operate on a cluster, so we may need to go there eventually.

Adding a new object will also simplify documentation for createInstance, (i.e. `$type Cluster[] $clusters <description>`), and the call itself.

You know more about Bigtable than I, so if there's a reason not to do this please let me know.

Example call to this method with a Cluster class:

```php
$bigtable->createInstance('my-instance', [
    'clusters' => [
        $bigtable->cluster(/** cluster args **/)
    ]
]);
```